internal/app/machined/pkg/controllers/network/etcfile.go

```diff
@@ -178,14 +178,14 @@ func (ctrl *EtcFileController) renderResolvConf(resolverStatus *network.Resolver
 
 var hostsTemplate = template.Must(template.New("hosts").Parse(strings.TrimSpace(`
 127.0.0.1       localhost
-{{ .IP }}       {{ .Hostname }} {{ if ne .Hostname .Alias }}{{ .Alias }}{{ end }}
+{{ printf "%-15s" .IP }} {{ .Hostname }} {{ if ne .Hostname .Alias }}{{ .Alias }}{{ end }}
```

Comment:
this probably only handles the case with IPv4 addresses though? 

we could also use https://pkg.go.dev/text/tabwriter to produce nice looking file always?

Reply:
I refactored to use `tabwriter`